Makefile

```diff
@@ -35,3 +35,16 @@ checkstyle: black lint
 
 .PHONY: test
 test: only-test checkstyle
+
+venv = qem-bot
+environment:
```

Comment:
```suggestion
.PHONY: environment
environment:
```

Reply:
Any particular reason for the preference?

Reply:
well, because the target is not a file "environment" but an abstract target, same as in other ones